src/validate.ts

```diff
@@ -9,7 +9,16 @@ const SYNC_RESPONSE_SCHEMA = require('../intents/sync.response.schema.json');
 const QUERY_RESPONSE_SCHEMA = require('../intents/query.response.schema.json');
 const EXECUTE_RESPONSE_SCHEMA = require('../intents/execute.response.schema.json');
 const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schema.json');
+const ON_OFF_STATES_SCHEMA = require('../traits/onoff.states.schema.json');
+const ON_OFF_ATTRIBUTES_SCHEMA = require('../traits/onoff.attributes.schema.json');
 
+const COMMAND_STATES_EXPECT = {
+  'action.devices.commands.OnOff': ON_OFF_STATES_SCHEMA,
```

Comment:
Do you need access to the other schema files so you can fully validate your examples?

Reply:
Other two needed would be 'action.devices.traits.Brightness' and 'action.devices.traits.ColorSetting'. Because of the amount of time left for the internship Nick and I discussed about leaving these two schemas out for now. (@Fleker if you have any other thoughts on this)